frontend/package.json

```diff
@@ -50,7 +50,7 @@
     "build": "react-scripts build",
     "postbuild": "echo 'pushstate: enabled' > build/Staticfile",
     "lint": "npm-run-all -p lint:eslint lint:prettier",
-    "lint:eslint": "eslint --quiet --ext .js --ext .jsx ./src",
+    "lint:eslint": "eslint --quiet --ext .js --ext .jsx --ext .tsx ./src",
```

Comment:
we should add `--ext .ts` to this as well

Reply:
Should we do the TS upgrade in this PR?

Reply:
I'd suggest no, I had a few speed bumps doing this in a previous project so I'd just merge this as-is and we can make a ticket to do the upgrade and add `--ext .ts`

Reply:
I have a separate branch already pending with this change, but it didn't seem like a good idea to make this depend on that. :-)